.gitignore

```diff
@@ -96,7 +92,4 @@ src/scaffolding.config
 
 # JetBrains Rider
 .idea/
-*.sln.iml
-src/Integration/Directory.Build.props
-
-PromotedSqlScripts/
```

Comment:
This is still needed https://github.com/Particular/NServiceBus.Persistence.Sql/blob/master/src/Integration/Shared/SqlPersistenceSettings.cs#L8 

Do I add a nested .gitignore like we discussed @bording @ramonsmits ?